packages/airnode-node/src/workers/local-gateways/validation.ts

```diff
@@ -250,7 +250,11 @@ export function verifySignOevDataRequest(
     };
   }
 
-  return { success: true, validUpdateValues: map(validDecodedBeacons, 'decodedValue') };
+  return {
+    success: true,
+    validUpdateValues: map(validDecodedBeacons, 'decodedValue'),
```

Comment:
Wouldn't it be better to return `validDecodedBeacons` ?

Reply:
I was thinking about it and decided against it since we only really need two things. If we end up needing/returning more I'll refactor it. It's a detail.